src/ui/flutter_app/lib/widgets/game_page.dart

```diff
@@ -53,16 +53,17 @@ class GamePage extends StatefulWidget {
   static double screenPaddingH = AppTheme.boardScreenPaddingH;
 
   final EngineType engineType;
-  final Engine engine;
 
-  GamePage(this.engineType) : engine = NativeEngine();
+  const GamePage(this.engineType);
 
   @override
   _GamePageState createState() => _GamePageState();
 }
 
 class _GamePageState extends State<GamePage>
     with RouteAware, SingleTickerProviderStateMixin {
+  final Engine _engine = NativeEngine();
```

Comment:
I don't know if this is a problem Lint found or if you found it yourself? It looks like this fix is very advanced.